From ce9cb0aced6e8e1510ef6c557268aa97406a94ea Mon Sep 17 00:00:00 2001
From: Alex Kwiatkowski <alex+git@fremantle.io>
Date: Wed, 20 Nov 2024 20:57:00 -0800
Subject: [PATCH] use PATH echo

---
 Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 4825648..f047842 100644
--- a/Makefile
+++ b/Makefile
@@ -90,7 +90,7 @@ install:
 # Output .gdbinit instructions only if DESTDIR is empty.
 # When non-empty, this is almost certainly part of a package building exercise,
 # and the message is irrelevant and confusing.
-	@if [ X"$(SKIP_NOTE)" = X"" ] ; then /bin/echo -e "\033[0;33m" ; fi
+	@if [ X"$(SKIP_NOTE)" = X"" ] ; then echo -e "\033[0;33m" ; fi
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then echo "In order for gdb to be able to load the Python script that displays COBOL variables," ; fi
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then echo "you will have to create a ~/.gdbinit file that has the following lines in it:" ; fi
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then echo "" ; fi
@@ -106,7 +106,7 @@ install:
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then 	echo "to create a sample file (in this folder) with that content." ; fi
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then 	echo "" ; fi
 	@if [ X"$(SKIP_NOTE)" = X"" ] ; then 	echo "See https://sourceware.org/gdb/onlinedocs/gdb/Auto_002dloading-safe-path.html for more information" ; fi
-	@if [ X"$(SKIP_NOTE)" = X"" ] ; then 	/bin/echo -e "\033[0m" ; fi
+	@if [ X"$(SKIP_NOTE)" = X"" ] ; then 	echo -e "\033[0m" ; fi
 
 .PHONY : uninstall
 uninstall:
-- 
2.46.0

